bokeh/models/tickers.py

```diff
@@ -66,7 +66,7 @@ class AdaptiveTicker(Ticker):
     The smallest allowable interval between two adjacent ticks.
     """)
 
-    max_interval = Float(100.0, help="""
+    max_interval = Float(float('Inf'), help="""
```

Comment:
If it worked with infinity before, then I wouldn't touch the subject in this PR.